Automattic/wp-calypso
# Revamp Blaze banner

Related to https://jira.tumblr.net/browse/SELFSERVE-411

## Description
We want to revamp the banner according to a [Figma](https://www.figma.com/file/uvh2KtY65b7hGBHI9exbJ1/BlazePress?node-id=5448-44706&t=VXEiEfmXBAA0C9O1-0) design.

## Proposed Changes
* Change layout for promote posts / campaigns only
* Revamp a banner

## Testing Instructions



* Checkout this branch in you local environment
* Run Calypso (`yarn && yarn start`)
* Make sure you have less than 3 campaigns, otherwise you need to hack this [condition](https://github.com/Automattic/wp-calypso/pull/76094/files#diff-af45fef67dabb389ee3b79b06fd228c64fa0c2e369993c26b3e105c2b1e2cc04R277)
* Ensure you see the update banner
* Try out both posts and campaigns pages and make sure they look fine (I didn't make it responsive in a decent way, so it will be done down the road)

## Screenshots

### Banner is revamped
![blaze-press-new-banner](https://user-images.githubusercontent.com/115007291/233693848-8a5f88a4-f72b-44eb-addf-67f88abf68c4.gif)

### Updated Layout
![blaze-press-new-layout](https://user-images.githubusercontent.com/115007291/233693924-0563e223-d6aa-4f96-b7ed-1f3551a9cbc8.gif)

## Pre-merge Checklist
